css-brochure-form/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-align: center;
+  text-transform: uppercase;
+  letter-spacing: 0.03em;
+  color: #e4561e;
+}
+
+body {
+  width: 550px;
+  height: 500px;
+  margin: 100px;
+}
```

Comment:
You don't want to affect the body element like this. The `body` is meant to be everything in the viewport, so constricting its size is a no-no. You should apply this to the `.container` element instead